bittensor/utils/async_substrate.py

```diff
@@ -74,17 +402,187 @@ async def get_storage_item(self, module: str, storage_function: str):
         storage_item = metadata_pallet.get_storage_function(storage_function)
         return storage_item
 
+    def _get_current_block_hash(self, block_hash: Optional[str], reuse: bool):
+        return block_hash if block_hash else (self.last_block_hash if reuse else None)
+
+    async def _init_runtime(
+        self, block_hash: Optional[str] = None, block_id: Optional[int] = None
+    ) -> Runtime:
+        """
+        Not currently working.
```

Comment:
Still not working?

Reply:
No. This may be implemented in the future, but for the moment, we utilise `init_runtime`, rather than this `_init_runtime`. I would like to see this eventually get finished, but it is not currently doable with the time constraints.

Reply:
Perhaps adding TODO will make this more obvious. 
You can close/solve this comment after adding TODO or without it (it's up to you)